wtransport/Cargo.toml

```diff
@@ -35,7 +35,7 @@ rustls-pemfile = "1.0.2"
 socket2 = "0.5.3"
 thiserror = "1.0.40"
 time = { version = "0.3.21", optional = true }
-tokio = { version = "1.28.1", default-features = false, features = ["macros", "fs"] }
+tokio = { version = "1.28.1", default-features = false, features = ["macros", "fs", "rt-multi-thread"] }
```

Comment:
I would suggest to not touch this line, as `wtransport` does not need tokio runtime. That's just the example.

Could you please instead add the dependency like:
```
[dev-dependencies]
hyper = ...
tokio = { version = "1.28.1", features = ["rt-multi-thread"] }
```

Reply:
Sure!

Reply:
I pushed the changes :)

Reply:
Thanks

Reply:
Welcome 😄